diff --git a/node_modules/@payloadcms/richtext-lexical/dist/features/toolbars/shared/ToolbarDropdown/index.js b/node_modules/@payloadcms/richtext-lexical/dist/features/toolbars/shared/ToolbarDropdown/index.js
index d961986..c377689 100644
--- a/node_modules/@payloadcms/richtext-lexical/dist/features/toolbars/shared/ToolbarDropdown/index.js
+++ b/node_modules/@payloadcms/richtext-lexical/dist/features/toolbars/shared/ToolbarDropdown/index.js
@@ -121,6 +121,7 @@ export const ToolbarDropdown = ({
     Icon: Icon,
     itemsContainerClassNames: [`${baseClass}-items`, ...(itemsContainerClassNames || [])],
     label: label,
+    stopCloseOnClickSelf: true,
     children: renderedItems
   }, groupKey);
 };
